py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -80,23 +81,45 @@ def index(
     source_ir: Optional[SourceIR],
     name: str,
     input: TRTTensor,
-    index: Union[TRTTensor, Sequence[TRTTensor]],
+    index: Union[
+        TRTTensor,
+        Sequence[TRTTensor],
+        np.ndarray,
+        Sequence[np.ndarray],
+        torch.Tensor,
+        Sequence[torch.Tensor],
+    ],
 ) -> TRTTensor:
     adv_indx_indices = []
     tensor_indices = []
     # _LOGGER.debug(f"The index shape is {index.shape}")
     # check if the input is dynamic
     dynamic_shape = has_dynamic_shape(input.shape)
-
+    # is_numpy is a flag to specify if all the indices are numpy or torchTensor.
+    # If any is not this flag will be set to False
+    is_numpy = True
```

Comment:
This line can be replaced if the above suggestion is used

Reply:
I meant the line `is_numpy = True` can be replaced with `is_numpy = all(isinstance(ind, (torch.Tensor, np.ndarray)) for ind in index if index is not None)`

Reply:
The `is_numpy = True` line would no longer be needed